src/store/index.ts

```diff
@@ -112,7 +114,8 @@ export function getVuexStore () {
       mutateOfficeAddresses,
       mutateOrgPersonList,
       mutateShareClasses,
-      mutateUserEmail
+      mutateUserEmail,
+      mutateOriginalIA
```

Comment:
I've been trying to keep this alphabetized :)

Reply:
oops, will fix in next commit!